src/H5detect.c

```diff
@@ -22,28 +22,26 @@ static const char *FileHeader = "\n\
  * distribution tree, or in https://www.hdfgroup.org/licenses.               *\n\
  * If you do not have access to either file, you may request a copy from     *\n\
  * help@hdfgroup.org.                                                        *\n\
- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *";
+ *                                                                           *\n\
```

Comment:
Perhaps Github is presenting this in an unusual manner, but I'm not certain that you wanted two '*' on this line?

Reply:
I'll check it out. I recall it looking fine when I inspected the output.